deepchem/next.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('next').NextConfig} */
+
```

Comment:
Add a line to the README explaining changes for github pages

Reply:
Added necessary documentation.